docs/wiki/The CI environment - Publishing.md

```diff
@@ -1,6 +1,7 @@
 This section provides an overview of the principles the publishing is built upon, how it is set up, and how you can interact with it.
```

Comment:
Regarding relocation:
- Move to 'Solution Creation' after 'Orchestration' section

Reply:
done